cli/medperf/tests/commands/auth/test_password_change.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from unittest.mock import call, ANY
+
+from medperf.comms import Comms
+from medperf.commands.auth import PasswordChange
+from medperf.utils import storage_path
+from medperf import config
+
```

Comment:
Possible to add a failure test that checks that the token is NOT removed if change password fails?

Reply:
Added, thanks!